packages/cli/src/ActiveWorkflowRunner.ts

```diff
@@ -757,6 +741,7 @@ export class ActiveWorkflowRunner {
 	 */
 	// TODO: this should happen in a transaction
 	async remove(workflowId: string) {
+		if (!this.orchestrationService.isLeader) return;
```

Comment:
Should we instead send a message to the current leader? Asking because the public API calls this method and if we handle it here, as well as the `add` method we can simplify things.

Reply:
Tracking all calls to `remove` I can see it being used by the public API as well as source control, user management (when deleting a user) so it'd be great if we could centralize it.